docs/python-installation.md

```diff
@@ -110,26 +110,14 @@ The following environment variables need to be configured when instrumenting you
 | `$SCOPE_REPOSITORY`  | Autodetected (\*) | Repository URL to use when sending data to Scope                  |
 | `$SCOPE_SOURCE_ROOT` | Autodetected (\*) | Repository root path                                              |
 
-The following optional parameters can also be configured:
-
-| Environment variable | Default value | Description                                    |
-| -------------------- | ------------- | ---------------------------------------------- |
-| `$SCOPE_SERVICE`     | `default`     | Service name to use when sending data to Scope |
```

Comment:
why is this removed?

Reply:
It should have been  moved to the configuration section